magento/pwa-studio
# [Doc] Production launch checklist

## Description

Creates a topic listing important items to check prior to a production launch.

## Related Issue

Closes PWA-310

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
1. Run the linter: `yarn lint src/tutorials/pwa-studio-fundamentals/production-launch-checklist`
1. Verify no linting issues
1. Start the preview server: `yarn develop`
1. Navigate to the tutorials section
1. Verify topic is listed on the left navigation and linked to the correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
